gradlew

```diff
@@ -231,4 +231,16 @@ eval "set -- $(
         tr '\n' ' '
     )" '"$@"'
 
```

Comment:
I was originally thinking "Wow this will be so easy, just check if the $project.name is a connector, and if so include it" using the variables here: https://stackoverflow.com/questions/29368201/get-project-name-of-task-in-gradle

But then I realized `settings.gradle.` gets evaluated before any actual steps begin to run, so there's not any real way to use built-in gradle variables to achieve conditional project inclusions. So I got a bit creative and added a regex arg matcher against which searches for the `:airbyte-integrations:connectors:...` pattern, which by the looks of this repo, is very uniform and dependable. 

`settings.gradle` _can_ use environment variables, so if a connector project is passed, we just set an appropriate env variable for it. Otherwise do nothing.